openshift/templates/aries-mediator-agent/aries-mediator-agent-deploy.yaml

```diff
@@ -183,6 +183,11 @@ objects:
                   --wallet-storage-config "$(eval echo \"${WALLET_STORAGE_CONFIGURATION}\")"
                   --wallet-storage-creds "$(eval echo \"${WALLET_STORAGE_CREDENTIALS}\")"
                   --admin 0.0.0.0 ${MEDIATOR_AGENT_HTTP_ADMIN_PORT}
+
+                  # ToDo:
+                  #   - Figure out how to get this to work with PQ and non-PQ instances.
```

Comment:
I think this is one of those things where we should have a Helm chart with configuration setting, so that parts of the template can be turned on/off selectively.

Alternatively, we could look into Kustomize, however based on the fact that having a redistributable chart would help us with other services that depend on ACA-Py and it would be good for the community and adoption, I am leaning towards this option.